### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by joel-molero**

### CODE REVIEW

File: staff/joel-molero/cv-styled/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<body>
+    <header> 
+        <div class="child">
+        <h1 class="nombre">Joel Molero</h1>
+        </div>
+        <br>
+        <address>
+            Address: Spain, Barcelona, Febot street, Nº24, 2-1<br>
+            Tel: 93 710 63 28 || Mobile: 601 72 34 21<br>
+            Email: joel.molero@gmail.com<br>
```
Comment (aleixcam) [inline staff/joel-molero/cv-styled/index.html]: Use p tags instead of br

File: staff/joel-molero/cv-styled/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            <h1 class="title"><u>My Working Experience</u></h1>
+        <section class="working">
+            <br>
+            
+            <div class="workingExperience1">
+                <h3>2003-2007</h3>
+                <p class="experience">Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam nam, at repellendus, enim quis voluptates facilis, in vitae earum natus ex est ipsam sunt ullam delectus eius maxime tempore nihil!</p>
+            </div>
+            <div class="workingExperience2">
+                <h3>2008-2014</h3>
```
Comment (aleixcam) [inline staff/joel-molero/cv-styled/index.html]: use the ```<time>``` tag for dates

File: staff/joel-molero/cv-styled/styles.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+address {
+    display: block;
+    color: black;
+    background-color:#f9c4aa; 
+    border: 2px solid rgb(139, 133, 133);
+    text-shadow: 1px 0 0 rgb(255,255,255), -1px 0 0 rgb(255,255,255), 0 1px 0 rgb(255,255,255), 0 -1px 0 rgb(255,255,255), 0.5px 0.5px rgb(255,255,255), -0.5px -0.5px 0 rgb(255,255,255), 0.5px -0.5px 0 rgb(255,255,255), -0.5px 0.5px 0 #fff;
+}
+
+
```
Comment (aleixcam) [inline staff/joel-molero/cv-styled/styles.css]: too many newlines

File: staff/joel-molero/cv-styled/index.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    <br>
+    </header>
+    <main>
+            <h1 class="title"><u>My Working Experience</u></h1>
+        <section class="working">
+            <br>
+            
+            <div class="workingExperience1">
+                <h3>2003-2007</h3>
+                <p class="experience">Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam nam, at repellendus, enim quis voluptates facilis, in vitae earum natus ex est ipsam sunt ullam delectus eius maxime tempore nihil!</p>
```
Comment (pepdbm7) [inline staff/joel-molero/cv-styled/index.html]: this class is not used in the css file, maybe it's not necessary to define it

Comment (joel-molero) [reply staff/joel-molero/cv-styled/index.html]: Fixed

**Pull request merged**